client/src/core/ErrorBoundary.js

```diff
@@ -65,7 +65,7 @@ export class ErrorBoundary extends React.Component {
       testing_for_stale_client: false,
     });
 
-    throw this.state.error;
+    throw new Error(this.state.error);
```

Comment:
Wait I think this one is throwing Error(Error())? But I don't think that's a big deal

Reply:
👍